### PR TITLE
Add weather sensors to Homematic IP

### PR DESCRIPTION
**Pull request by SukramJ**
**Description:**
Add weather sensors to Homematic IP

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21887

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/homematicip_cloud.markdown [hunk 1]
```diff
@@ -28,6 +29,7 @@ redirect_from:
   - /components/light.homematicip_cloud/
   - /components/sensor.homematicip_cloud/
   - /components/switch.homematicip_cloud/
+  - /components/weather.homematicip_cloud/
```
Comment (klaasnicolaas) [inline source/_components/homematicip_cloud.markdown]: this is not necessary

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/homematicip_cloud.markdown (hunk 1 above)
Comment (SukramJ) [reply source/_components/homematicip_cloud.markdown]: Removed line 20 and 32

Comment (klaasnicolaas) [reply source/_components/homematicip_cloud.markdown]: You can keep line 20 but the redirects are for existing pages.

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ Approved, can be merged as soon te parent PR is merged

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @SukramJ! 👍

**Pull request merged**